src/Core/Vipr.Core/CodeModel/OdcmMethod.cs

```diff
@@ -8,6 +8,8 @@ namespace Vipr.Core.CodeModel
 {
     public class OdcmMethod : OdcmAnnotatedObject
     {
+        public string Namespace { get; set; }
```

Comment:
This property is being set in the PR, but it is not being read anywhere. What, then, is the purpose of the property?